datafusion/core/benches/parquet_statistic.rs

```diff
@@ -82,11 +103,18 @@ fn create_parquet_file(dtype: TestTypes, row_groups: usize) -> NamedTempFile {
     for _ in 0..row_groups {
         let batch = match dtype {
             TestTypes::UInt64 => make_uint64_batch(),
+            TestTypes::Int64 => make_int64_batch(),
             TestTypes::F64 => make_f64_batch(),
             TestTypes::String => make_string_batch(),
             TestTypes::Dictionary => make_dict_batch(),
         };
-        writer.write(&batch).unwrap();
+        if data_page_row_count_limit.is_some() {
```

Comment:
Perhaps we can add a comment here explaining why we write a batch at a time (basically because the page limit is only checked on record batch boundaries